### ISSUE TITLE
`repo` scope is not enough to run the script -> authorization error

### ISSUE THREAD

**Issue by jugmac00** [nedbat/dinghy#2]
> To run dinghy you will need a GitHub [personal access token](https://github.com/settings/tokens) with the correct scopes, probably "repo"

Nope. I get `"Unauthorized. You need to create a GITHUB_TOKEN environment variable."` then.

For a quick test run I checked "most" boxes during the token creation, and then the script works. So it is probably something in between.   I have no time to investigate myself right now.

**Comment by nedbat** [comment]
I've been trying to test scenarios to understand what is needed.  I found that "repo" was enough unless I was looking at an organization project, then I needed "read:org" scope.   Can you tell me what URL or configuration you were using? I also thought the error message you showed would only happen with no token at all, but GitHub's handling of authorization problems seems ad-hoc.

**Comment by jugmac00** [comment]
Hi Ned, sorry for not adding a reproducer - I was wrongly thought this is a common enough problem.

```
(venv) $ source .env 
(venv) $ echo $GITHUB_TOKEN
xxx
(venv) $ python -m dinghy https://github.com/jugmac00/Flask-Reuploaded
error: dinghy error: Unauthorized. You need to create a GITHUB_TOKEN environment variable.
```

I even went back in browser history to make sure I ticked the correct check boxes...
![Screenshot from 2022-03-13 21-11-43](https://user-images.githubusercontent.com/9895620/158077283-e4cedbcd-1dd6-4f62-9ff7-5cf4dc6be06c.png)

As I said - the next token with almost full permissions worked.

I just created another token, again only with `repo` permissions and now it works.

So this was either a bogus token, a temporary glitch in the auth system, or a layer 8 problem :-)

Anyway, sorry for the noise - great tool!
